### DISCUSSION TITLE
Working on "success" variant on toast messages

### DISCUSSION THREAD

**Discussion by anshumalivfx** [shadcn-ui/ui#4326] [Show and tell]
What you guys think ? 
or
do you have any suggestions ?

<img width="458" alt="Screenshot 2024-07-18 at 13 23 18" src="https://github.com/user-attachments/assets/1468e39c-270a-4bf9-b8f5-2db79d39f41c">

**Comment by anshumalivfx** [comment]
Also we can make "warning" variant aswell

**Reply by arhamkhnz** [reply]
I think adding a "warning" variant sounds good! Are you working on it already? I'm also planning to jump on this.

**Reply by anshumalivfx** [reply]
Once I get some onClick time off work I will get back to this
